### PR TITLE
SUIT-16501 - new test operation

### PR DESCRIPTION
**Pull request by AntonSBilan**

### CODE REVIEW

**Review by JeKachan — APPROVED** [review]

File: packages/translate/lib/testLine.tsx [hunk 1]
```diff
@@ -432,6 +437,30 @@ const translateBrowserCommandTestLine = (
...
+	const status = testLine.excluded ? 'excluded' : lineResult?.result;
+	const condition = testLine.condition
+		? translateCondition(testLine.condition, false, appConfig, elements, lineResult)
+		: undefined;
+
+	switch (testLine.deviceSettings.type) {
+		case 'setOrientation':
+			const orientation = deviceOrientationsMap[testLine.deviceSettings.params.orientation];
+
+			return <test-line title={<text>Set screen orientation to {orientation}</text>} status={status}>
```
Comment (andriichumak) [inline packages/translate/lib/testLine.tsx:455]: Perhaps we could put some formatting on orientation? Nice to have

Comment (andriichumak) [reply packages/translate/lib/testLine.tsx:455]: @AntonSBilan , I think `<input>` would fit nicely here.

File: packages/translate/lib/testLine.tsx [hunk 2]
```diff
@@ -432,6 +437,32 @@ const translateBrowserCommandTestLine = (
...
+	const condition = testLine.condition
+		? translateCondition(testLine.condition, false, appConfig, elements, lineResult)
+		: undefined;
+
+	switch (testLine.deviceSettings.type) {
+		case 'setOrientation':
+			const orientation = deviceOrientationsMap[testLine.deviceSettings.params.orientation];
+
+			return <test-line title={
+				<fragment><text>Set screen orientation to</text> <input>{orientation}</input></fragment>
```
Comment (andriichumak) [inline packages/translate/lib/testLine.tsx:456]: Do we actually need `<text>` here? I'm not entirely sure, but it should work the same without it:
`<fragment>Set screen orientation <input>{orientation}</input></fragment>`

Comment (AntonSBilan) [reply packages/translate/lib/testLine.tsx:456]: I will check

Comment (AntonSBilan) [reply packages/translate/lib/testLine.tsx]: you are right, uodated

**Review by andriichumak — APPROVED** [review]

**Pull request merged**